.swiftlint.yml

```diff
@@ -51,3 +51,11 @@ custom_rules:
     message: Rule IDs must be all lowercase, snake case and not end with `rule`
     regex: identifier:\s*("\w+_rule"|"\S*[^a-z_]\S*")
     severity: error
+  fatal_error:
+    name: Fatal Error
+    excluded: "Tests/*"
+    message: Prefer using `queuedFatalError` over `fatalError`
+    regex: fatalError
```

Comment:
Shouldn't this be `'\bfatalError\b'`? Otherwise `someotherfatalError(...)` would trigger?